templates/myaccount/dashboard.php

```diff
@@ -35,8 +35,12 @@
 
 <p>
 	<?php
+	$dashboard_desc = __( 'From your account dashboard you can view your <a href="%1$s">recent orders</a>, manage your <a href="%2$s">billing address</a>, and <a href="%3$s">edit your password and account details</a>.', 'woocommerce' );
+	if ( get_option( 'woocommerce_ship_to_countries' ) !== 'disabled' ) {
```

Comment:
This option isn't the only way to disable shipping.
Please replace it to:

```php
if ( wc_shipping_enabled() ) {
```

Reply:
fe3de4f02cdf5f86f3c14469fe963b2df6e34a05